universe/fgnames/cities.txt

```diff
@@ -1,3421 +1,3421 @@
 Aachen, Germany 
 Aalborg, Denmark 
 Aalst, Belgium 
-Aarhus, Denmark
+Aarhus, Denmark		
 Abbotsford, BC, Canada
-Aberdeen, NC 
+Aberdeen, NC 	
 Aberdeen, Scotland, U.K.
-Aberdeen, WA 
+Aberdeen, WA 	
 Aberystwyth, Wales, U.K.
-Abilene, TX 
-Abingdon, VA 
-Abington, MA 
-Abington, PA 
+Abilene, TX 	
+Abingdon, VA 	
+Abington, MA 	
+Abington, PA 	
 Abu Dhabi, AD, United Arab Emirates
-Acapulco, Mexico
-Accra, Ghana
-Achim, Germany
-Acton, MA 
-Acworth, GA 
-Ada, OK 
-Adams, MA 
+Acapulco, Mexico		
+Accra, Ghana		
+Achim, Germany		
+Acton, MA 	
+Acworth, GA 	
+Ada, OK 	
+Adams, MA 	
 Addis Ababa, Ethiopia
-Adel, IA 
+Adel, IA 	
 Adelaide, Australia
-Adelfia, Italy
-Aden, Yemen
-Adrian, MI 
-Agana, Guam 
+Adelfia, Italy		
+Aden, Yemen		
+Adrian, MI 	
+Agana, Guam 	
 Aguascalientes, Mexico
-Aiken, SC 
+Aiken, SC 	
 Air Force Academy, CO
 Ajdovscina, Slovenia
-Akron, OH 
+Akron, OH 	
 Al Kharj, Saudi Arabia
 Al-'Ain, A.D., United Arab Emirates
-Alameda, CA 
-Alamogordo, NM 
-Alamosa, CO 
-Albacete, Spain
-Albany, Australia 
-Albany, GA 
-Albany, NY 
-Albuquerque, NM 
+Alameda, CA 	
+Alamogordo, NM 	
+Alamosa, CO 	
+Albacete, Spain		
+Albany, Australia 	
+Albany, GA 	
+Albany, NY 	
+Albuquerque, NM 	
 Albury, NSW, Australia
-Alessandria, Italy
-Alexander City, AL 
-Alexandria, Egypt
-Alexandria, LA 
-Algiers, Algeria
+Alessandria, Italy		
+Alexander City, AL 	
+Alexandria, Egypt		
+Alexandria, LA 	
+Algiers, Algeria		
 Alice Springs, Australia
-Alingss, Sweden
-Allegan, MI 
-Allen Park, MI 
-Allen, TX 
-Allentown, PA 
-Alma, QU., Canada
-Almeria, Spain
-Alpena, MI 
-Altoona, PA 
-Alturas, CA 
-Alva, OK 
+Alingsås, Sweden		
```

Comment:
probably should get rid of the excess spaces

Reply:
Wow, there were even tabs, uploading a new version...